### PR TITLE
polish translation

### PR DESCRIPTION
**Pull request by xkotori**
Polish translation for Pubby

### CODE REVIEW

File: locales/pl-PL.json [hunk 1]
```diff
@@ -0,0 +1,645 @@
...
+    "ui_code": "pl-PL",
+    "validation_code": "pl",
+    "dayjs_code": "pl",
+    "name": "Polski",
+    "translated_by": "Translated by {users}",
+    "reviewed_by": "Reviewed by {users}"
+  },
+  "$contributors": {
+    "authors": [{ "name": "norelock" }],
+    "reviewers": [{ "name": "salomaosnff" }, { "name": "Pyrael" }]
```
Comment (salomaosnff) [inline locales/pl-PL.json:13]: Leave reviewers empty as you are the author of the file.

**Review by xkotori — COMMENTED** [review]
prob fixed removal of en-us

File: locales/pl-PL.json [hunk 2]
```diff
@@ -0,0 +1,645 @@
+{
+  "locale": {
+    "code": "pl-PL",
+    "ui_code": "pl-PL",
+    "validation_code": "pl",
+    "dayjs_code": "pl",
+    "name": "Polski",
+    "translated_by": "Translated by {users}",
```
Comment (salomaosnff) [inline locales/pl-PL.json:8]: Forgot to translate here

File: locales/pl-PL.json [hunk 3]
```diff
@@ -0,0 +1,645 @@
...
+      "user_ban": {
+        "title": "Zbanuj użytkownika",
+        "body": "Czy na pewno chcesz zablokować użytkownika {username} w pokoju?"
+      },
+      "user_kick": {
+        "title": "Wyrzuć użytkownika",
+        "body": "Czy na pewno chcesz wyrzucić użytkownika {username} z pokoju?"
+      },
+      "user_site_ban": {
+        "title": "Zbanuj użytkownika z serwisu,
```
Comment (salomaosnff) [inline locales/pl-PL.json:498]: Missing double quotes at the end of the line.

File: locales/en-us.json [hunk 4]
```diff
@@ -1,645 +0,0 @@
-{
```
Comment (salomaosnff) [inline locales/en-us.json:1]: You deleted the English translation file. Please put it again.

**Review by salomaosnff — CHANGES_REQUESTED** [review]
Just make these changes.

**Pull request closed without merging**